run/test/specs/disappearing_message_checks_android.spec.ts

```diff
@@ -0,0 +1,477 @@
+  //   );
+  // Wait for control messages to disappear before sending image (to check if the control messages are interfering with finding the untrusted attachment message)
+  await sleepFor(60000);
+  await device1.sendImage(platform, testMessage);
+  await device2.clickOnElementAll({
+    strategy: "accessibility id",
+    selector: "Untrusted attachment message",
+  });
+  await device2.clickOnElementAll({
+    strategy: "accessibility id",
+    selector: "Download media",
+  });
+  await sleepFor(60000);
+
+  await Promise.all([
+    device1.hasElementBeenDeletedNew({
+      strategy: "accessibility id",
+      selector: "Message body",
+      maxWait: 1000,
+      text: testMessage,
+    }),
+    device2.hasElementBeenDeletedNew({
+      strategy: "accessibility id",
+      selector: "Message body",
+      maxWait: 1000,
+      text: testMessage,
+    }),
+  ]);
+  await closeApp(device1, device2);
+}
+
+async function disappearingVideoMessage(platform: SupportedPlatformsType) {
+  const { device1, device2 } = await openAppTwoDevices(platform);
+  // Create user A and user B
+  const [userA, userB] = await Promise.all([
+    newUser(device1, "Alice", platform),
+    newUser(device2, "Bob", platform),
+  ]);
+  const time: DMTimeOption = "1 minute";
+  const mode: DisappearActions = "sent";
+  await newContact(platform, device1, userA, device2, userB);
+
+  await setDisappearingMessage(
+    platform,
+    device1,
+    ["1:1", "Disappear after send option", "1 minute"],
+    device2
+  );
+  // TODO FIX
+  // await device2.disappearingControlMessage(
+  //   `${userA.userName} has set messages to disappear ${time} after they have been ${mode}.`
+  // );
+  // await device2.disappearingControlMessage(
+  //   `You set messages to disappear ${time} after they have been ${mode}.`
+  // );
+  // Wait for control messages to disappear before sending image (to check if the control messages are interfering with finding the untrusted attachment message)
+  // await sleepFor(60000);
+  await device1.sendVideo(platform);
+  await device2.clickOnByAccessibilityID("Untrusted attachment message");
+  await device2.clickOnByAccessibilityID("Download media");
+  // Wait for disappearing message timer to remove video
+  await sleepFor(60000);
+  await Promise.all([
+    device1.hasElementBeenDeletedNew({
+      strategy: "accessibility id",
+      selector: "Media message",
+    }),
+    device2.hasElementBeenDeletedNew({
+      strategy: "accessibility id",
+      selector: "Media message",
+    }),
+  ]);
+  await closeApp(device1, device2);
+}
+
+async function disappearingVoiceMessage(platform: SupportedPlatformsType) {
+  const { device1, device2 } = await openAppTwoDevices(platform);
+  // Create user A and user B
+  const [userA, userB] = await Promise.all([
+    newUser(device1, "Alice", platform),
+    newUser(device2, "Bob", platform),
+  ]);
+  const time: DMTimeOption = "1 minute";
+  const mode: DisappearActions = "sent";
```

Comment:
`const time: DMTimeOption = "1 minute";` works, but my prefered way for those (unless if a let and not a const) is actually to do 
`const time = "1 minute" as const;`
this makes `time` type be a string which can only be "1 minute"
So when you try to call a function which takes an argument of type `DMTimeOption` which is the same as `"1 minute" | "5 minutes" ...` typescript is happy

ask me for details if needed

Reply:
this comment applies to the whole file (and maybe others, search and replace)